querybook/webapp/components/DataDocPythonCell/PythonCellResultView.tsx

```diff
@@ -0,0 +1,76 @@
+import JsonView from '@uiw/react-json-view';
+import { lightTheme } from '@uiw/react-json-view/light';
+import { vscodeTheme } from '@uiw/react-json-view/vscode';
+import React, { useCallback, useMemo } from 'react';
+
+import { StatementResultTable } from 'components/StatementResultTable/StatementResultTable';
+import { useShallowSelector } from 'hooks/redux/useShallowSelector';
+import { IStoreState } from 'redux/store/types';
+
+import './PythonCellResultView.scss';
+
+export const PythonCellResultView = ({ stdout, stderr }) => {
+    const theme = useShallowSelector(
+        (state: IStoreState) => state.user.computedSettings['theme']
+    );
+
+    const themeStyle = useMemo(
+        () => ({
+            ...(theme === 'dark' ? vscodeTheme : lightTheme),
+            '--w-rjv-background-color': 'var(--bg-lightest)',
+        }),
+        [theme]
+    );
+
+    const getTableData = useCallback((data) => {
+        const columns = data['columns'];
+        const records = data['records'];
+
+        return [
+            columns,
+            ...records.map((record) => columns.map((col) => record[col])),
+        ];
+    }, []);
+
+    const renderOutput = (output) => {
+        if (typeof output === 'object') {
```

Comment:
what if it is output object without a type key?

Reply:
it will fallback to the plain text view